contracts/network/BancorNetwork.sol

```diff
@@ -820,26 +818,17 @@ contract BancorNetwork is IBancorNetwork, Upgradeable, ReentrancyGuardUpgradeabl
     /**
      * @inheritdoc IBancorNetwork
      */
-    function withdrawNetworkFees(
-        address recipient
-    )
-        external
-        whenNotPaused
-        onlyRoleMember(ROLE_NETWORK_FEE_MANAGER)
-        validAddress(recipient)
-        nonReentrant
-        returns (uint256)
-    {
+    function burnNetworkFees() external whenNotPaused nonReentrant returns (uint256) {
         uint256 currentPendingNetworkFeeAmount = _pendingNetworkFeeAmount;
-        if (currentPendingNetworkFeeAmount == 0) {
+        if (currentPendingNetworkFeeAmount < _minNetworkFeeBurn) {
             return 0;
         }
 
         _pendingNetworkFeeAmount = 0;
 
-        _masterVault.withdrawFunds(Token(address(_bnt)), payable(recipient), currentPendingNetworkFeeAmount);
+        _masterVault.withdrawFunds(Token(address(_bnt)), payable(address(_bnt)), currentPendingNetworkFeeAmount);
```

Comment:
I'd add a comment here that this "transfer to the token" burns the tokens (I do remember this, but it always seems weird on the first sight)

Reply:
Added